src/qtiCommonRenderer/helpers/Graphic.js

```diff
@@ -337,6 +464,56 @@ const GraphicHelper = {
         return element;
     },
 
+    /**
+     * Create a new Element with another design (double border - black and white)
+     * @param {Raphael.Paper} paper - the interaction paper
+     * @param {String} type - the shape type
+     * @param {String|Array.<Number>} coords - qti coords as a string or an array of number
+     * @param {Object} [options] - additional creation options
+     * @param {String} [options.id] - to set the new element id
+     * @param {String} [options.style = basic] - to default style
+     * @param {Boolean} [options.qtiCoords = true] - if the coords are in QTI format
+     * @returns {Raphael.Element} the created element
+     */
+    createElement2: function (paper, type, coords, options) {
+        const self = this;
+        let groupEl;
+        const shaper = shapeMap[type] ? paper[shapeMap[type]] : paper[type];
+        const shapeCoords = options.qtiCoords !== false ? self.raphaelCoords(paper, type, coords) : coords;
+        const stateCls = options.style || 'basic';
+
+        if (typeof shaper === 'function') {
+            try {
+                let clipPathDefId;
+                if (clipPathDefSetter[type]) {
+                    clipPathDefId = clipPathDefSetter[type](paper, shapeCoords);
+                }
+
+                groupEl = paper.group({ class: `${ELEMENT_CLASS_NAME2} ${stateCls}` });
+
+                const innerEl = shaper.apply(paper, shapeCoords).attr({ class: 'hotspot2-inner' });
```

Comment:
Can the `-inner` / `-outer` classNames be extended from the `ELEMENT_CLASS_NAME` const?